### PR TITLE
Add aarch64 build

### PR DESCRIPTION
**Pull request by Tobias-Fischer**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by Tobias-Fischer** [comment]
@conda-forge-admin, please rerender

**Comment by Tobias-Fischer** [comment]
Needs https://github.com/conda-forge/libestools-feedstock/pull/4

**Comment by Tobias-Fischer** [comment]
@conda-forge-admin, please rerender

**Comment by Tobias-Fischer** [comment]
@conda-forge-admin, please rerender

**Comment by Tobias-Fischer** [comment]
Hi @peci1 - this is ready for review :)

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,14 +13,15 @@ build:
   run_exports:
     - {{ pin_subpackage('festival', max_pin='x.x') }}
   skip: True  # [not linux]
```
Comment (peci1) [inline recipe/meta.yaml:15]: Didn't you also want to enable osx?

**Review by peci1 — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (Tobias-Fischer) [reply recipe/meta.yaml:15]: Too hard for now unfortunately

**Review by peci1 — COMMENTED** [review]
Hmm, it seems the aarch64 build hasn't been activated by this PR...

**Comment by peci1** [comment]
Oh, now I see. The aarch64 build runs on Travis...

**Pull request merged**